rotkehlchen/chain/ethereum/modules/adex/adex.py

```diff
@@ -476,14 +469,12 @@ def _deserialize_channel_withdraw(
             ) from e
 
         try:
-            address = to_checksum_address(user_address)
+            address = deserialize_ethereum_address(user_address)
             identity_address = inverse_identity_address_map[address]
-            tx_address = to_checksum_address(tx_address)
-            token_address = to_checksum_address(token_address)
-        except (KeyError, ValueError) as e:
-            msg = str(e)
-            if isinstance(e, KeyError):
-                msg = f'Missing key in event: {msg}.'
+            tx_address = deserialize_ethereum_address(tx_address)
+            token_address = deserialize_ethereum_address(token_address)
+        except KeyError as e:
```

Comment:
Here you need to handle the `DeserializationError` that can be raised from `deserialize_ethereum_address`.

So basically what the code was doing was fine, but you need to replace `ValueError`with `DeserializationError`